data/abilities.ts

```diff
@@ -341,7 +341,8 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			if (!source || source === target || !target.hp || !move.totalDamage) return;
 			const lastAttackedBy = target.getLastAttackedBy();
 			if (!lastAttackedBy) return;
-			const damage = move.multihit ? move.totalDamage : lastAttackedBy.damage;
+			const darts = move.id === 'dragondarts' && source.getSmartTargets(target, move).length > 1;
```

Comment:
Using `getSmartTargets` here doesn't make sense; it's not necessarily going to give you the same result as when it was used before damage happened.